src/main/java/com/dev/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,24 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.dto.UserRegistrationDto;
+import com.dev.cinema.security.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
```

Comment:
```suggestion
    @PostMapping
```
for creating a new user let's user just post mapping without creating additional resource